pkg/jx/cmd/create_addon_flagger.go

```diff
@@ -103,5 +105,22 @@ func (o *CreateAddonFlaggerOptions) Run() error {
 	if err != nil {
 		return fmt.Errorf("Flagger Grafana deployment failed: %v", err)
 	}
+
+	// Enable Istio in production namespace
+	client, err := o.KubeClient()
+	if err != nil {
+		return fmt.Errorf("error enabling Istio in production namespace: %v", err)
+	}
+	var ns string
+	ns, err = o.findEnvironmentNamespace("production")
```

Comment:
Maybe provide this over a command argument, in case someone has a different name for production environment. This argument can default to `production`.